core/server/helpers/author.js

```diff
@@ -10,32 +10,31 @@
 // Block helper: `{{#author}}{{/author}}`
 // This is the default handlebars behaviour of dropping into the author object scope
 
-var hbs             = require('express-hbs'),
-    _               = require('lodash'),
-    utils          = require('../utils'),
-    localUtils     = require('./utils'),
-    author;
+var proxy = require('./proxy'),
+    _ = require('lodash'),
+    SafeString = proxy.SafeString,
+    helpers = proxy.hbs.handlebars.helpers,
+    templates = proxy.templates,
+    url = proxy.url;
 
-author = function (options) {
+module.exports = function author(options) {
     if (options.fn) {
-        return hbs.handlebars.helpers.with.call(this, this.author, options);
+        return helpers.with.call(this, this.author, options);
```

Comment:
The only feedback i can give is, that i find it a little bit less readable.
`helpers.with` vs `hbs.handlebars.helpers.with`